src/core/title.js

```diff
@@ -69,6 +75,19 @@ function setDocumentTitle(conf, h1Elem) {
     documentTitle = `Preview of PR #${conf.prNumber}: ${documentTitle}`;
   }
 
+  if (conf.hasOwnProperty("level")) {
+    if (validLevel(conf.level)) {
+      h1Elem.append(document.createTextNode(` Level ${conf.level}`));
```

Comment:
No need to create a text node... the .append() api does it for free! :) 
```suggestion
      h1Elem.append(` Level ${conf.level}`);
```

Reply:
Note, the word "level" might need to be localized. We can just have the English localization for now.

Reply:
haha braintwins. writing that in the level plugin now.